projects/cashmere/src/lib/drag-list/drag-list.component.html

```diff
@@ -13,69 +13,71 @@
                 <div class="preserve-whitespace" style="white-space: pre">{{ targetTooltip }}</div>
             </hc-pop>
         </div>
-        <div *ngFor="let assignment of _modifiedAssignments" class="hc-drag-list-assignments">
-            <div class="hc-drag-list-target-name" [title]="assignment.target[targetHoverTextField] ?? ''">
-                <hc-icon
-                    fontSet="fa"
-                    fontIcon="fa-check"
-                    *ngIf="!assignment.locked"
-                    [style.visibility]="!!assignment.assignment ? 'visible' : 'hidden'"
-                ></hc-icon>
-                <hc-icon fontSet="fa" fontIcon="fa-lock" [hcPop]="lockedText" trigger="hover" *ngIf="!!assignment.locked"></hc-icon>
-                <hc-pop #lockedText>{{ optionLockedText }}</hc-pop>
-                {{ assignment.target[targetDisplayNameField] }}
-            </div>
-
-            <div
-                class="hc-drag-list-assignment hc-drag-list-placeholder"
-                (dragenter)="_dragenter($event)"
-                (dragleave)="_dragleave($event)"
-                *ngIf="!assignment.assignment && !assignment.locked"
-                [attr.data-target-id]="assignment.target[targetIdField]"
-                (drop)="_makeAssignment($event)"
-                (dragover)="_dragover($event)"
-            >
-                {{ targetPlaceholder }}
-            </div>
-            <div
-                class="hc-drag-list-assignment"
-                (dragenter)="_dragenter($event)"
-                (dragleave)="_dragleave($event)"
-                *ngIf="!!assignment.assignment && !assignment.locked"
-                [attr.data-target-id]="assignment.target[targetIdField]"
-                [attr.data-option-id]="assignment.assignment[optionIdField]"
-                (drop)="_makeAssignment($event)"
-                (dragover)="_dragover($event)"
-            >
+        <div class="hc-drag-list-targets-container">
```

Comment:
The only actual change to this file was the addition of this wrapper div to allow the body of the list to scroll independently from the header.